src/isolate.rs

```diff
@@ -105,6 +109,19 @@ impl IsolateState {
     metrics.ops_completed += 1;
     metrics.bytes_received += bytes_received;
   }
+
+  pub fn start_repl(&self, name: String) {
+    let mut repls = self.repls.lock().unwrap();
+    match repls.entry(name.clone()) {
+      Occupied(entry) => entry.into_mut(),
+      Vacant(entry) => entry.insert(DenoRepl::new(&name, &self.dir)),
+    };
+  }
+
+  pub fn exit_repl(&self, name: String) {
+    let mut repls = self.repls.lock().unwrap();
+    repls.remove(&name).unwrap();
+  }
```

Comment:
I think the LineReader should be in the resources table. These functions should be moved/rewritten to be in the style of the resources table.